### PR TITLE
Add C# and VB QuickInfo integration tests

### PR DESCRIPTION
**Pull request by rchande**
Tag @dotnet/roslyn-ide for review
Will remove the XML-based test in a subsequent PR

### CODE REVIEW

**Comment by Pilchie** [comment]
Doh!  Signed off when the tests are failing :(

**Comment by rchande** [comment]
Sigh, passed on my machine :(

File: src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs [hunk 1]
```diff
@@ -255,6 +261,11 @@ protected void VerifyParameters(params (string name, string documentation)[] par
             }
         }
 
+        protected void VerifyQuickInfo(string expectedText)
+        {
```
Comment (tmeschter) [inline src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs]: Why does this method exist?

**Comment by rchande** [comment]
retest windows_debug_vs-integration_prtest

**Comment by rchande** [comment]
retest windows_debug_vs-integration_prtest please

**Comment by rchande** [comment]
retest windows_debug_vs-integration_prtest

**Comment by rchande** [comment]
retest this please

**Comment by rchande** [comment]
retest this please

**Comment by rchande** [comment]
@tmeschter `VerifyQuickInfo` method removed.

**Comment by rchande** [comment]
Looks like the failures are in BasicWinForms and unrelated.

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpQuickInfo.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    {
+        protected override string LanguageName => LanguageNames.CSharp;
+
+        public CSharpQuickInfo(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(CSharpQuickInfo))
+        {
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public void QuickInfo_MetadataDocumentation()
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpQuickInfo.cs]: "QuickInfo_" prefix is probably unnecessary.

File: src/VisualStudio/IntegrationTest/TestUtilities/InProcess/QuickInfoToStringConverter.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            {
+                return textBlock.Text;
+            }
+
+            var sb = new StringBuilder();
+            BuildStringFromInlineCollection(textBlock.Inlines, sb);
+            return sb.ToString();
+        }
+
+        private static void BuildStringFromInlineCollection(InlineCollection inlines, StringBuilder sb)
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/TestUtilities/InProcess/QuickInfoToStringConverter.cs]: Inline into the other function? Or is this called from more than one place?

**Comment by jasonmalinowski** [comment]
@tmeschter: still blocking on this one?

**Comment by rchande** [comment]
retest windows_debug_vs-integration_prtest

**Pull request merged**